php/commands/src/Alias_Command.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+use Mustangostang\Spyc;
+
+/**
+ * Retrieves and sets alias for WP Installs.
+ *
+ * Aliases are shorthand references to WordPress installs. For instance,
+ * `@dev` could refer to a development install and `@prod` could refer to
+ * a production install. This command gives you visibility in what
+ * registered aliases you have available.
+ *
+ * ## EXAMPLES
+ *
+ *     # Get alias information.
+ *     $ wp cli alias get @dev
+ *     ssh: dev@somedeve.env:12345/home/dev/
+ *
+ *     # Add alias.
+ *     $ wp cli alias add prod login@host /path/to/wordpress/install/
+ *     Success: Added '@prod' alias.
+ *
+ *     # Update alias.
+ *     $ wp cli alias update @prod newlogin@host /updated/wordpress/path/
+ *     Success: Updated 'prod' alias.
+ *
+ *     # Delete alias.
+ *     $ wp cli alias delete @prod
+ *     Success: Deleted '@prod' alias.
+ *
+ * @package wp-cli
+ * @when before_wp_load
+ */
+class Alias_Command extends WP_CLI_Command {
+
+	/**
+	 * List available WP-CLI aliases.
+	 *
+	 * Aliases are shorthand references to WordPress installs. For instance,
+	 * `@dev` could refer to a development install and `@prod` could refer to
+	 * a production install. This command gives you visibility in what
+	 * registered aliases you have available.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--format=<format>]
+	 * : Render output in a particular format.
+	 * ---
+	 * default: yaml
+	 * options:
+	 *   - yaml
+	 *   - json
+	 *   - var_export
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # List all available aliases.
+	 *     $ wp cli alias list
+	 *     ---
+	 *     @all: Run command against every registered alias.
+	 *     @prod:
+	 *       ssh: runcommand@runcommand.io~/webapps/production
+	 *     @dev:
+	 *       ssh: vagrant@192.168.50.10/srv/www/runcommand.dev
+	 *     @both:
+	 *       - @prod
+	 *       - @dev
+	 *
+	 * @subcommand list
+	 */
+	public function list_( $args, $assoc_args ) {
+		WP_CLI::print_value( WP_CLI::get_runner()->aliases, $assoc_args );
+	}
+
+	/**
+	 * Gets the value for an alias.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Key for the alias.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Get alias.
+	 *     $ wp cli alias get @prod
+	 *     ssh: dev@somedeve.env:12345/home/dev/
+	 *
+	 * @subcommand get
+	 */
+	public function get( $args, $assoc_args ) {
+		list( $alias ) = $args;
+		$aliases       = Spyc::YAMLLoad( WP_CLI::get_runner()->get_global_config_path() );
```

Comment:
Why are you loading the aliases directly here? This code seems to break in certain contexts, as the global config path might be modified (for Behat for example).

Why not just use the following?
```suggestion
		$aliases       = WP_CLI::get_runner()->aliases;
```

Reply:
In Behat, you can see that there is actually no global config: https://travis-ci.org/wp-cli/wp-cli/jobs/439284200#L612

When you create a new file `wp-cli.yml` in the WP folder, that would actually be a project config, so you could try with the `get_project_config_path()` method instead.

Reply:
No, `get_project_config_path()` would actually be wrong as well, you need the merged version, and need to make changes to the global one by default and maybe the project one when provided with a flag.